### PR TITLE
Make provider lookup case insensitive and update CosmosDB api version

### PR DESCRIPTION
**Pull request by stuartleeks**
- update swagger config to pick up latest Cosmos DB version
 - update api version lookup to be case-insensitive on resource type (fixes issue preventing CosmosDB node expanding)
 - update swagger-update to use `https://` repo address rather than `git@...` repo address to work with anonymous access

### CODE REVIEW

File: scripts/swagger_update/app.py [hunk 1]
```diff
@@ -29,6 +29,9 @@
         # storage package-2019-06 references privatelinks.json which is not listed in the input files
         "storage": { 
             "package-2019-06": ["./Microsoft.Storage/stable/2019-06-01/privatelinks.json"] 
+        },
+        "cosmos-db" : {
+            "package-2020-04" : ["./Microsoft.DocumentDB/stable/2019-08-01/cosmos-db.json"]
```
Comment (lawrencegripper) [inline scripts/swagger_update/app.py:34]: wooo I did something useful!

**Review by lawrencegripper — APPROVED** [review]
LGTM

File: scripts/swagger_update/app.py (hunk 1 above)
Comment (stuartleeks) [reply scripts/swagger_update/app.py:34]: YES - very much YES!

Comment (stuartleeks) [reply scripts/swagger_update/app.py:34]: Seeing that made me happy as I was working on this 😄

**Pull request merged**